CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ### vNext
+- `apollo-engine-reporting`: BREAKING CHANGE: By default, send no GraphQL variable values to Apollo's servers instead of sending all variable values. Use the new EngineReportingOption `sendVariableValues` to send some or all variable values, possibly after transforming them. 
```

Comment:
Yeah I think Adam's right: this should explicitly say that this is a breaking change only if you don't currently specify `privateVariables` at all, and to preserve the old default, you'll want to pass `new ApolloServer({engine: {sendVariableValues: {sendAll: true}}})`.  (I do think it's good to put it in the context of `new ApolloServer` since that's where people will actually write it.)

Hmm, is it odd that the top option name starts with `send` and some but not all of the sub-option names do? Should they be `all` and `none` instead?

Reply:
I'll update the CHANGELOG to be more specific! Do you think it would be worth noting this in the docs specific to the deprecated options (`privateVariables`, `privateHeaders`) as well?

I think since the new option names already begin with `send...`, that the suboptions `all` and `none`  should be clear enough. Will also make that change!